public/components/common/restart-cluster-manager-callout.tsx

```diff
@@ -65,9 +65,8 @@ class WzRestartClusterManagerCallout extends Component<IWzRestartClusterManagerC
     try{
       this.setState({ warningRestarting: true, warningRestartModalVisible: false});
       const data = await restartClusterOrManager(this.props.updateWazuhNotReadyYet);
-      // this.setState({ warningRestarting: false });
       this.props.onRestarted();
-      this.showToast('success', `${data.restarted === 'cluster' ? 'Restarting cluster, it will take up to 30 seconds.': 'Manager was restarted'}`)
+      this.showToast('success', `${data.restarted === 'cluster' ? 'Cluster was restarted': 'Manager was restarted'}`)
```

Comment:
If the possible values of `data.restarted` are `cluster` or `manager` change the string template to:

```ts
`${data.restarted} was restarted`
```